modules/serverless-logic-job-service-leader-election.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-job-service.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-job-service-leader-election_{context}"]
+= Job service leader election process
+
+The Job service operates as a singleton service, meaning only one active instance can schedule and execute jobs.
+
+To prevent conflicts when deployed in the cloud, where multiple instances may be running, the Job service supports a leader election process. Only the instance elected as the leader manages external communication to receive and schedule jobs.
+
+Instances that are not leaders remain inactive in a waiting state and continuously attempt to become the leader. When a new instance starts, it does not assume leadership immediately but begins the process to become the leader.
```

Comment:
```suggestion
Non-leader instances remain inactive in a standby state but continue attempting to become the leader through the election process. When a new instance starts, it does not immediately assume leadership. Instead, it enters the leader election process to determine if it can take over the role.
```

Reply:
After this part we could explain the new mechanism to  limit the amount of time to get the leader status, introducing the configuration parameter `kogito.jobs-service.management.leader-check.expiration-in-seconds`.

@wmedvede wdyt?

Reply:
I think that now, with the reduced version we can document that property when we show the custom execution section.
Operator managed deployments are already setting this value.